### DISCUSSION TITLE
First time users install errors

### DISCUSSION THREAD

**Discussion by bananacy** [mediacms-io/mediacms#774] [Q&A]
Hello,

Edited, i fixed the issue. delete this.

/J

**Comment by mgogoulos** [comment]
What has been the issue? Some times it helps other even if it's something you consider noone else will make!

**Comment by bananacy** [comment]
I had tried to install it on arm64 server at hetzner, which required it'd own docker compose version, but it still wouldn't work properly.  Installed it on intel server and it works fine.


Very good platform but would like to see  larger images (less columns of frontend) and translation (to greek), and colour customization.


Excellent work though!

23 May 2023 12:02:17 Markos Gogoulos ***@***.***>:

**Comment by mgogoulos** [comment]
Thanks for the kind words! I remember there was a discussion about running it on ARM (https://github.com/mediacms-io/mediacms/discussions/653)

These are good improvements that you mention and there has been a plan to work on all of them (specially translation) however it has gotten stalled due to lack of time! But they will all find their way some time soon.
